proto/autoops/service.proto

```diff
@@ -160,6 +161,67 @@ message ListWebhooksResponse {
   int64 total_count = 3;
 }
 
+message CreateProgressiveRolloutRequest {
+  string environment_namespace = 1;
+  CreateProgressiveRolloutCommand command = 2;
+}
+
+message CreateProgressiveRolloutResponse {}
+
+message GetProgressiveRolloutRequest {
+  string id = 1;
+  string environment_namespace = 2;
+}
+
+message GetProgressiveRolloutResponse {
+  ProgressiveRollout progressive_rollout = 1;
+}
+
+message DeleteProgressiveRolloutRequest {
+  string id = 1;
+  string environment_namespace = 2;
+  DeleteProgressiveRolloutCommand command = 3;
+}
+
+message DeleteProgressiveRolloutResponse {}
+
+message ListProgressiveRolloutRequest {
+  enum OrderBy {
+    DEFAULT = 0;
+    CREATED_AT = 1;
+    UPDATED_AT = 2;
+  }
+  enum OrderDirection {
+    ASC = 0;
+    DESC = 1;
+  }
+  string environment_namespace = 1;
+  int64 page_size = 2;
+  string cursor = 3;
+  repeated string feature_ids = 4;
+  OrderBy order_by = 5;
+  OrderDirection order_direction = 6;
+  optional ProgressiveRollout.Status status = 7;
+  string search_keyword = 8;
+  optional ProgressiveRollout.Type type = 9;
+}
+
+message ListProgressiveRolloutResponse {
+  repeated ProgressiveRollout progressive_rollouts = 1;
+  string cursor = 2;
+}
+
+message ExecuteProgressiveRolloutRequest {
+  string environment_namespace = 1;
+  string id = 2;
+  ChangeProgressiveRolloutTriggeredAtCommand
```

Comment:
I didn't understand what is this command for.
Can you describe how the `time` will be used on the server?

Reply:
@cre8ivejp 

I added the comment. Does it make sense?

Reply:
The executor changes the `triggered_at` in the auto ops rules because it has that property in the proto message. But the progressive rollout doesn't have a `triggered_at` field.
So it's hard to understand what this command is for.

Reply:
Talked at discord.